### ISSUE TITLE
Atari 7800 db9

### ISSUE THREAD

**Issue by Pepeart** [MiSTer-DB9/Issues#15]
Hola,
El core del Atari 2600 se retiro por que el nuevo core del Atari 7800 es mejor y tambien corre los juegos del Atari 2600. Podrian release el Core del Atari 7800 con db9 para poder usar los controles originales? Gracias

**Comment by HerrBerzerk** [comment]
YES, please support the original Joysticks and Paddle Controllers with DB9. It is no fun to play old Atari Games with new Playstation Controllers.

Atari had so many great Paddle Games like Kaboom, Circus, or Warlords with 4 Players simultaneously. Star Raider with 1 Joystick and one Keypad or the great Indy500 with the "Spinner Controllers". Can I support someone at Patreon to make this happen?
